### DISCUSSION TITLE
GitHub Pages SSL Certificate Expiration

### DISCUSSION THREAD

**Discussion by vrfm-chevalier** [community/community#22398] [New to GitHub]
<p>Hello,</p>
<p>I have a GitHub Page with a custom domain, that stopped working today, with error <strong>NET::ERR_CERT_DATE_INVALID</strong>.</p>
<p>It appears the SSL certificate has expired with this error in the browser. How do I regenerate this SSL cert? I could find nothing in the GitHub docs about this.</p>
<hr>
<p>Subject: <a href="http://www.vrfm.io" rel="noopener nofollow ugc">www.vrfm.io</a></p>
<p>Issuer: Let’s Encrypt Authority X3</p>
<p>Expires on: Jan 6, 2021</p>
<p>Current date: Jan 6, 2021</p>
<p>PEM encoded chain:<br>
…</p>
<hr>
<p>The repository itself (<strong>vrfm-chevalier/vrfm-chevalier.github.io</strong>) still functions as normal.</p>
<p>Any suggestions on how to fix this certificate error?</p>
<p>Thanks</p>

**Comment by AlfredoRamos** [comment]
<p>You might want to reach the support team, since SSL certificates are handled by GitHub itself.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="16" height="16">
      <a href="https://support.github.com/request" target="_blank" rel="noopener nofollow ugc">support.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://support.github.com/request" target="_blank" rel="noopener nofollow ugc">GitHub Support</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>There’s a section specifically for HTTPS certificates issues.</p>
<p>You could also use services like CloudFlare.</p>

**Comment by vrfm-chevalier** [comment]
<p>Thanks. I sent a help request already.</p>
<p>The Settings section of the repo has been showing this for about 24 hours now. So I assume it’s a server-side error on their part.</p>
<hr>
<p>TLS certificate is being provisioned. This may take up to 15 minutes to complete.</p>
<p>1 of 3</p>
<p>Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient.</p>
<hr>

**Comment by vrfm-chevalier** [comment, marked as answer]
<p>I got a reply from a very helpful support rep. It turns out ALL CNAME records should point to the repo and you should not chain CNAME calls. e.g. don’t CNAME your www to your apex domain. Instead CNAME to your repo just like your apex entry. The redirects prevent certificate generation.</p>

**Comment by APB-LDN** [comment]
<p>Thanks for the info. So does that mean my CNAME should only include the repo url? As in ‘‘<a href="https://github.com/APB-LDN/APB-LDN.github.io" class="inline-onebox" rel="noopener nofollow ugc">GitHub - APB-LDN/APB-LDN.github.io: Personal website of Arthur P.B. LAUDRAIN</a>’’? It should not include the custom domain name?</p>

**Comment by sweeden-ttu** [comment]
I don't understand the solution to this problem?   How do you fix an expired certificate?

**Reply by APB-LDN** [reply]
It's not about fixing an expired certificate per se. It's about allowing automated certificate renewal. The way I had things set up in my CNAME records meant the process wouldn't complete.
